FlowCrypt/src/test/java/com/flowcrypt/email/security/pgp/PgpMsgTest.kt

```diff
@@ -0,0 +1,679 @@
+/*
+ * © 2016-present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com
+ * Contributors:
+ *     Ivan Pizhenko
+ */
+
+package com.flowcrypt.email.security.pgp
+
+import java.net.URLDecoder
+import org.junit.Test
+import org.junit.Assert.assertTrue
+import org.pgpainless.util.Passphrase
+import java.nio.charset.Charset
+
+class PgpMsgTest {
+
+  private data class MessageInfo(
+      val content: List<String>,
+      val quoted: Boolean? = null,
+      val params: String,
+      val charset: String = "UTF-8"
+  )
+
+  companion object {
+    private val privateKeys = listOf(
+        PgpMsg.SecretKeyInfo(
+            passphrase = "flowcrypt compatibility tests".toCharArray(),
+            armored = "-----BEGIN PGP PRIVATE KEY BLOCK-----\r\n" +
```

Comment:
@tomholub @IvanPizhenko @seisvelas I'm thinking about the code style here. As for me, it's not so good to store such a long string here. It bloats the code and makes it less readable. I don't need to know the whole content of a private key or an encrypted message to analyze tests. Instead, now I have to do a lot of scrolling. In my opinion, it'll be better to store such data in resources.

Reply:
I have no preference either way. By putting them here, you have to scroll but it will never get out of sync. Maybe by putting in resources you could later delete the test but leave the resource still there? So we may accumulate resources that are unused. But it makes tests indeed cleaner and more readable.

On Android, we should just follow whatever you have already been doing in other tests.

Reply:
will move to resources